CHANGELOG.rst

```diff
@@ -0,0 +1,67 @@
+0.8.0 - assert consistency
+--------------------------
+
+Initial release.
+
+This is the first release that can be used in production
```

Comment:
i think this intro should link to the countermitm section here early on, also to introduce the whole concept.